src/helpers/BoxHelper.js

```diff
@@ -83,7 +84,7 @@ BoxHelper.prototype.update = ( function () {
 
 		position.needsUpdate = true;
 
-		this.geometry.computeBoundingSphere();
```

Comment:
I expect it is wise to update the bounding sphere whenever a geometry's vertices are modified. Whatever we do, it should be consistent.